cebra/datasets/hippocampus.py

```diff
@@ -56,10 +79,19 @@ class SingleRatDataset(cebra.data.SingleSessionDataset):
 
     """
 
-    def __init__(self, name="achilles", root=_DEFAULT_DATADIR):
+    def __init__(self, name="achilles", root=_DEFAULT_DATADIR, download = True):
         super().__init__()
```

Comment:
Because we need to do this for every dataset, lets add this to the baseclass (or a Mixin, both fine for me). I would make the following work:

```suggestion
    def __init__(self, name="achilles", root=_DEFAULT_DATADIR, download = True):
        super().__init__(
            download = download,
            data_url = rat_dataset_urls[name]["url"],
            data_checksum = rat_dataset_urls[name]["checksum"]
        )
```

such that every dataset simply tells the base class what its URL is. The code from line 88--92 is then the same across every dataset.